### PR TITLE
feat(relevant reps): date check to show rr or not

### PR DESCRIPTION
**Pull request by jonathangoulding**

### CODE REVIEW

File: packages/forms-web-app/src/pages/projects/relevant-representations/representations.js [hunk 1]
```diff
@@ -70,7 +71,8 @@ exports.getRepresentations = async (req, res) => {
 		pageOptions,
 		searchTerm,
 		queryUrl,
-		commentsTypeFilterItems
+		commentsTypeFilterItems,
+		showReps: isDateAfterTodaysDate(applicationResponse.DateRRepAppearOnWebsite)
```
Comment (Philip-Philippou1) [inline packages/forms-web-app/src/pages/projects/relevant-representations/representations.js:75]: I think we may have to handle the case where the date is not set.  I think .DateRRepAppearOnWebsite  gets set just before they publish the reps so can this handle null or not set?

Comment (jonathangoulding) [reply packages/forms-web-app/src/pages/projects/relevant-representations/representations.js:75]: date not set will result in an invalid date and be false.

Comment (jonathangoulding) [reply packages/forms-web-app/src/pages/projects/relevant-representations/representations.js:75]: i'll add a test for the case.

**Review by Philip-Philippou1 — APPROVED** [review]

**Pull request merged**